### ISSUE TITLE
Possibility of auto indices refresh

### ISSUE THREAD

**Issue by czende** [BitBagCommerce/SyliusElasticsearchPlugin#3]
I would suggest to have option to choose between auto refresh and manually refresh indices. And have it in the readme :) I couldn't imagine that in production you will trigger that console command manually. or with cron script. FOSElasticaBundle has great option to set up all doctrine events 
```
user:
   persistence:
       driver: orm #the driver can be orm, mongodb or phpcr
       model: Application\UserBundle\Entity\User
       listener: ~ # by default, listens to "insert", "update" and "delete"
```
or just pick events you want
```
persistence:
     listener:
        insert: true
        update: false
        delete: true
```

**Comment by bitbager** [comment]
I think that we can mention it in the `README.md` that this option could be overridden 🙂

**Comment by bitbager** [comment]
Closed in #1a7e3ef.
